### PR TITLE
Update access-cluster.md

### PR DESCRIPTION
**Pull request by guzj11**
sync with english version.

### CODE REVIEW

File: content/zh/docs/tasks/access-application-cluster/access-cluster.md [hunk 1]
```diff
@@ -469,16 +469,15 @@ This shows the proxy-verb URL for accessing each service.
...
 -->
 这展示了访问每个服务的 proxy-verb URL。
 例如，如果集群启动了集群级别的日志（使用 Elasticsearch），并且传递合适的凭证，
 那么可以通过
 `https://104.197.5.247/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/`
 进行访问。日志也能通过 kubectl 代理获取，例如：
 `http://localhost:8080/api/v1/namespaces/kube-system/services/elasticsearch-logging/proxy/`。
-（参阅 [上面的内容](#accessing-the-cluster-api) 来获取如何使用 kubectl 代理来传递凭证）
-
+（参阅 [使用 Kubernetes API 访问集群](/zh/docs/tasks/administer-cluster/access-cluster-api/) 来获取如何使用 kubectl 代理来传递凭证）
```
Comment (tengqm) [inline content/zh/docs/tasks/access-application-cluster/access-cluster.md:480]:
```suggestion
（参阅[使用 Kubernetes API 访问集群](/zh/docs/tasks/administer-cluster/access-cluster-api/)
了解如何传递凭据，或者使用 kubectl proxy）
```

Comment (guzj11) [reply content/zh/docs/tasks/access-application-cluster/access-cluster.md]: suggestion applied

**Comment by guzj11** [comment]
> 
> 
> /lgtm

can we merge this commit?

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**